### DISCUSSION TITLE
Can @internationalized/date add helpers for startOfDay and endOfDay?

### DISCUSSION THREAD

**Discussion by rostero1** [adobe/react-spectrum#5517] [Q&A]
And what about setHours or setMinutes, etc...?

**Comment by devongovett** [comment]
You can use the `set` method for this:

```js
date.set({hour: 0, minute: 0, second: 0, millisecond: 0})
```

https://react-spectrum.adobe.com/internationalized/date/ZonedDateTime.html#setting-fields
